### ISSUE TITLE
Spurious warning with untyped multiple declarator

### ISSUE THREAD

**Issue by dalcinl** [MarcoGorelli/cython-lint#89]
#### Reproducer

```
$ echo 'cdef a, b, c' > /tmp/tmp.pyx
$ cython-lint /tmp/tmp.pyx 
/tmp/tmp.pyx:1:5: comma after base type in definition
```

**Comment by MarcoGorelli** [comment]
isn't this correct? shouldn't you set a type there, like `cdef int a, b, c`?

**Comment by dalcinl** [comment]
AFAIU, in this case Cython implicitly assumes the type is `object`. I have a few of these cases in mpi4py since the dawn of Cython, this probably originated in Pyrex.

**Comment by dalcinl** [comment]
BTW, if you point me in the right direction, maybe I would be able to sketch a PR.

**Comment by MarcoGorelli** [comment]
I'd rather keep this one, thanks, it's helped find some real issues. You can specify object explicitly

**Comment by MarcoGorelli** [comment]
closing then, but thanks for the suggestion
